kube_operator/client.go

```diff
@@ -130,11 +125,29 @@ func (c KubeClient) Install(tar string, envVars map[string]string, agId string)
 	// Sort the k8s api objects by kind
 	apiObjMap := sortAPIObjects(k8sObjs)
 
+	nsObj, namespace, err := getOperatorNamespace(apiObjMap)
+	if err != nil {
+		return err
+	}
+
+	if nsObj != nil {
+		glog.V(3).Infof(kwlog(fmt.Sprintf("attempting to create namespace %v", nsObj)))
+		_, err := c.Client.CoreV1().Namespaces().Create(nsObj)
+		if err != nil {
+			glog.V(3).Infof("Failed to create namspace %s. Continuing with installation.", nsObj.ObjectMeta.Name)
```

Comment:
Change the Infof to Warningf and use the kwlog function.

Reply:
Fixed